components/tools/samlsso-validator/org.wso2.carbon.identity.tools.saml.validator.ui/src/main/resources/web/tools-saml/saml_response_builder.jsp

```diff
@@ -115,7 +116,7 @@
 							<div style="width: 100%;">
 								<div style="width: 100%;"><textarea type="text"
 								name="samlResponse" id="samlResponse" class="text-box-big"
-								style="width: 99%; height: 250px; background: #FFFFFF;" autocomplete="off" readonly><%=responseDTO.getEncodedResponse()%></textarea></div>
+								style="width: 99%; height: 250px; background: #FFFFFF;" autocomplete="off" readonly><%=Encode.forHtml(responseDTO.getEncodedResponse())%></textarea></div>
```

Comment:
encoding forHtmlContent is enough here since used within a text area. Please test further if there could be any effect on encoding base64 encoded string for HTML. If it's not necessary there's no need to encode here